portal-web/test/functional/com/liferay/portalweb/paths/pathlib/uisections/Treeview.path

```diff
@@ -37,7 +37,7 @@
 </tr>
 <tr>
 	<td>NODE_SELECTED</td>
-	<td>//div[contains(@class,'card-type-directory') and contains(@class,'selected')]/div[contains(@class,'card-horizontal')]//span[contains(.,'${key_nodeName}')] | //div[contains(@role,'treeitem') and contains(@class,'active')]//div[contains(@class,'component-text')][normalize-space(text())='${key_nodeName}']</td>
+	<td>xpath=(//div[contains(@role,'treeitem') and contains(@class,'collapsed')] | //div[contains(@role,'treeitem') and contains(@class,'active')])//div[contains(@class,'component-text')][normalize-space(text())='${key_nodeName}'] | //div[contains(@class,'card-type-directory') and contains(@class,'selected')]/div[contains(@class,'card-horizontal')]//span[contains(.,'${key_nodeName}')]</td>
```

Comment:
Why all of these three locators need update? Since only one locator should be used in this case.